src/flag_gems/ops/pow_scalar.py

```diff
@@ -0,0 +1,47 @@
+import torch
```

Comment:
all forms of function pow are collected in pow.py . other files could be deleted.